nodes/arabica-devnet.md

```diff
@@ -50,9 +50,27 @@ like to run. Each node’s guide will link to the relevant network in
 order to show you how to connect to them. Learn about the different
 endpoint types [in the Cosmos SDK documentation](https://docs.cosmos.network/v0.50/learn/advanced/grpc_rest).
 
-### RPC endpoints
+### Production RPC endpoints
+
+<!-- markdownlint-disable MD013 -->
+<!-- markdownlint-disable MD034 -->
+
+These RPC providers are meant to be used in production environments.
+
+| Provider | URL |
+|--------|--------|
+| NewMetric | https://app.newmetric.xyz/start |
+| Numia | For RPC access: https://docs.numia.xyz/overview/rpc-api-access |
+| Numia | For data warehouse access: https://docs.numia.xyz/overview/sql-access/chains/celestia |
```

Comment:
I'd also add grove.city as they support consensus nodes with at least an RPC (not grpc though)

Reply:
good catch